R/tdata.R

```diff
@@ -0,0 +1,171 @@
+#' Create a new `tdata` object which contains
```

Comment:
The title of the manual for tdata is _**Create a new `tdata` object which contains**_.
It felt incorrect.

Along with the description, maybe we can do something like this?
```
#' Create a `tdata` Object
#'
#' Create a new object called `tdata` which contains `data`, a `reactive` list of data.frames
#' (or `MultiAssayExperiment`), with attributes:
#' \itemize{
#'   \item{`code` (`reactive`) containing code used to generate the data}
#'   \item{join_keys (`JoinKeys`) containing the relationships between the data}
#'   \item{metadata (`named list`) containing any metadata associated with the data frames}
#' }
```

It looks better in the manual page:
![image](https://user-images.githubusercontent.com/8597300/199843591-99503ef0-6d60-4973-a233-cf769ba6d49b.png)

Reply:
yup - done